00_hello_world/main_test.go

```diff
@@ -8,17 +8,47 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestMainOutput(t *testing.T) {
+func TestMain(t *testing.T) {
 	// Given
 	r := require.New(t)
 	var buf bytes.Buffer
 	out = &buf
 
-	// When
+	//When
 	main()
 
-	// Then
-	expected := strconv.Quote("Hallo du schöne Welt!\n")
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
 	actual := strconv.Quote(buf.String())
```

Comment:
you probably don't need `strconv.Quote` here

Reply:
Removed `strconv.Quote`